bigbluebutton-html5/imports/api/question-quiz/server/handlers/userTypedResponse.js

```diff
@@ -0,0 +1,34 @@
+import { check } from 'meteor/check';
+import QuestionQuizs from '/imports/api/question-quiz';
+import RedisPubSub from '/imports/startup/server/redis';
+
+export default function userTypedResponse({ header, body }) {
+  const REDIS_CONFIG = Meteor.settings.private.redis;
+  const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
+  const EVENT_NAME = 'RespondToQuestionQuizReqMsg';
+
+  const { questionQuizId, userId, answer } = body;
+  const { meetingId } = header;
+
+  check(questionQuizId, String);
+  check(meetingId, String);
+  check(userId, String);
+  check(answer, String);
+
+  const questionQuiz = QuestionQuizs.findOne({ meetingId, id: questionQuizId });
+
+  let answerId = 0;
+  questionQuiz.answers.forEach((a) => {
```

Comment:
You can change it to an [Array.indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)

Reply:
It was a unnecessary file which I kept in case of further additions in this feature. I removed it.